components/chat/ChatThread.tsx

```
@@ -77,6 +78,62 @@ export default function ChatThread({ messages, currentUserId, loading }: ChatThr
+                    }
+
+                    if (att.fileType === "video") {
+                      return (
+                        <video
+                          key={`${message._id}-att-${index}`}
+                          src={att.url}
+                          controls
+                          className="max-h-56 w-full rounded-md"
+                        >
+                          Your browser does not support video playback.
+                        </video>
```

Biome (2.4.4)

[error] Provide a track for captions when using audio or video elements.

(lint/a11y/useMediaCaption)